### PR TITLE
Illustration

### PR DESCRIPTION
**Pull request by jamesgorrie**
Adding Illustrator as a usage rights category.
Once we have the list we can also add parsers.

- [x] Put illustrator list in once we have it

### CODE REVIEW

File: metadata-editor/app/model/UsageRightsProperty.scala [hunk 1]
```diff
@@ -72,4 +79,12 @@ object UsageRightsProperty {
 
     case _ => List()
   }
+
+  private def illustrationProperties(u: UsageRights) = u match {
+    case _:ContractIllustrator     => List(
+      UsageRightsProperty("creator", "Illustrator", "string", true, Some(sortList(commissionedIllustrators))))
```
Comment (theefer) [inline metadata-editor/app/model/UsageRightsProperty.scala]: I'm confused by the `ContractIllustrator` class gets `commissionedIllustrators`?

**Comment by theefer** [comment]
Minor comment, else :+1:

File: metadata-editor/app/model/UsageRightsProperty.scala (hunk 1 above)
Comment (jamesgorrie) [inline metadata-editor/app/model/UsageRightsProperty.scala]: Your confusion is my stupidity, we would only have a list of contract illutrators.

**Comment by jamesgorrie** [comment]
- [ ] Update Composer to pick up on this automatically.

**Comment by paperboyo** [comment]
> Update Composer to pick up on this automatically.

Nice! What’s with Screenshots as they currently live outside Illustrations? (There might have been a semi-agreement that they should appear as “Illustrations” in Composer/frontends).

**Comment by jamesgorrie** [comment]
> Screenshots

Screengrabs category.

**Pull request merged**